vagga.yaml

```diff
@@ -75,7 +75,7 @@ commands:
       app: !Command
         container: django
         environ:
-          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/test
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/mainDB
```

Comment:
It will not work if I move it to container:
`Is the server running on host "127.0.0.1" and accepting
	TCP/IP connections on port 8458?
`